cmd/gossamer/account_test.go

```diff
@@ -95,7 +108,11 @@ func TestAccountGenerateType(t *testing.T) {
 func TestAccountImport(t *testing.T) {
 	testDir := utils.NewTestDir(t)
 	defer utils.RemoveTestDir(t)
-
+	directory := fmt.Sprintf("--basepath=%s", testDir)
+	err := app.Run([]string{"irrelevant", "account", directory, "--import"})
```

Comment:
you need to provide a value for `--import`, previously it was `testfile`

Reply:
@noot Where is the testfile sourced?